packages/react-core/src/components/Modal/examples/ModalWithDropdown.tsx

```diff
@@ -45,6 +47,16 @@ export const ModalWithDropdown: React.FunctionComponent = () => {
     }
   };
 
+  const getAppendLocation = () => {
+    // document doesn't exist when PatternFly website docs framework gets pre-rendered
+    // this is just to avoid that issue - works fine at runtime without it
+    if (typeof document !== 'undefined' && document.getElementById) {
+      return document.getElementById('modal-box-body-with-dropdown');
```

Comment:
Just as a relation to my above comment, updating these lines to:

```suggestion
    if (typeof document !== 'undefined' && document.querySelector) {
      return document.querySelector('.pf-v6-l-bullseye');
```

Should result in behavior consumers might expect/want (the menu not creating a Modal scrollbar).

Going this far, though, maybe it'd be useful to add props within the ModalContent component to allow adding a unique ID to the FocusTrap wrapper that gets the bullseye class applied.